### ISSUE TITLE
Logokat rotalni

### ISSUE THREAD

**Issue by lptr** [lptr/sarok.org#7]
Jelenleg az Apache access log meg talan mas logok is limit nelkul novekszenek, ami nem jo. Ezt meg kene oldani, valoszinu valami sima logrotate boven eleg.

**Comment by lberki** [comment]
Melyik logok? Ahogy neztem, a randa 10 gigas mailbox mellett ket dolog van, ami jelentos helyet foglal:

* `/var/www/sarok3/www/userimages` -- 1.3 giga kep, ami sok, de szuksegesnek latszik.
* `/var/www/lib/mysql/sarok` -- ahogy nezem, a harom nagy tabla a `mail`, `entries` meg `comments`, es ugyan nem neztem a formatumot, de egyik se tunik trivialisan legyakhatonak.

**Comment by lptr** [comment]
Access log meg error log – ezekbol kb. ket gigat toroltunk Dimivel. A user images meg a mail nem novekszik read-only modban, a logok igen, szoval ezert kene megoldani.

**Comment by lptr** [comment]
Ja, most latom, hogy mitol van a mail aktivitas... Egen, azt is meg kene oldani. :)

**Comment by sarokuser1** [comment]
Well, tegnap reggel még SOKKAL több levél jött. De igen, az SQL hibák is érdekesek :D

**Comment by pppq** [comment]
A `log.txt` ismét nem megnyitható, lehet, hogy megint felfúvódtak ezek a fájlok.

**Comment by sarokuser1** [comment]
Almost! A `/var/log/sarok3` nem írható a httpd által, úgyhogy nem bírt logokat írni. chown után kiderült, hogy a MySQL nem üzemelt valamiért. Újraindítottam, most mety.
